cypress.json

```diff
@@ -7,5 +7,7 @@
     "runMode": 2,
     "openMode": 0
   },
-  "blockHosts": "*.google-analytics.com"
+  "blockHosts": "*.google-analytics.com",
+  "testFiles": ["**/**/*.spec.ts"],
```

Comment:
Let's tell cypress which files to include here to make our test script a little more readable.